transport/api/src/lib.rs

```diff
@@ -7,6 +7,7 @@ mod p2p;
 mod processes;
 mod timer;
 
+#[derive(Clone)]
 pub enum TransportOutput {
```

Comment:
Documentation on the type wouldn't hurt, even though it is almost self-explanatory